src/apis/mentorPost.js

```diff
@@ -0,0 +1,37 @@
+import { instance } from "./instance";
+import { mockResponse, postsData } from "../components/mentoring/mock";
+
+export async function getPostsReq(category, search, page) {
+  // return await instance.get(
+  //   `/mentorings/post?category=${category}&search=${search}&page=${page}`
+  // );
+
+  // api 구현 전까지 mock 데이터 반환
+  return page < 2 ? mockResponse(postsData) : mockResponse({ response: [] });
+}
+
+export async function getPostReq(pid) {
+  return await instance.get(`/mentorings/post/${pid}`);
+}
+
+export async function addPostReq(data) {
+  const { datas } = data;
+  return await instance.post("/mentorings/post", {
+    datas: datas,
```

Comment:
```suggestion
    datas,
```

객체 리터럴 생성시 key와 value의 변수명이 동일한 경우에는 축약 가능합니다.
다만 data 자체가 복수형(plural)이기 때문에 datas는 옳지 않은 key이긴 한데요,
서버에서 저렇게 api 명세를 정의한 상황이라면 어쩔 수 없겠네요.ㅎㅎ